### DISCUSSION TITLE
How to retrieve traces via grpc

### DISCUSSION THREAD

**Discussion by m1-s** [jaegertracing/jaeger#4385] [Q&A]
I am exploring the grpc API:
```bash
$ grpcurl -plaintext localhost:16685 list
                                        
grpc.health.v1.Health
grpc.reflection.v1alpha.ServerReflection
jaeger.api_v2.QueryService
jaeger.api_v2.metrics.MetricsQueryService
jaeger.api_v3.QueryService
```

Next I want to explore functions of the QueryService but receive error:
```
$ grpcurl -plaintext localhost:16685 list jaeger.api_v3.QueryService

Failed to list methods for service "jaeger.api_v3.QueryService": Symbol not found: jaeger.api_v3.QueryService
```

**Comment by yurishkuro** [comment]
This is a documented issue with the reflection API: https://www.jaegertracing.io/docs/1.44/apis/

Cf https://github.com/jaegertracing/jaeger/issues/3417

**Reply by chris-aeviator** [reply]
how are we supposed to query traces ?

**Reply by yurishkuro** [reply]
you can implement a grpc client using the schema we provide, just like you'd do with any other grpc service.
